docs/user_guides/tls_proxy.md

```diff
@@ -1,12 +1,12 @@
 # How to enable use TLS proxy
 
-Milvus proxy uses TLS mutual authentication. 
+Milvus proxy uses TLS mutual authentication.
 
 
 
 ## How to create your own certificate.
 
-### 1. Make sure openssl is installed. 
+### 1. Make sure openssl is installed.
```

Comment:
### Prerequisites

Make sure OpenSSL is installed. If you have not installed it, [build and install](https://github.com/openssl/openssl/blob/master/INSTALL.md) OpenSSL first.